### ISSUE TITLE
Fails when the browser is set to a zoom level different than 100%

### ISSUE THREAD

**Issue by tin-cat** [tin-cat/jquery-mosaic#7]

**Comment by tin-cat** [comment]
The new version 0.15 implements a workaround for a floating point-related bug that may solve this issue, but I haven't tested it yet.

**Comment by tin-cat** [comment]
Tested and working! The recent 0.15 version that addressed a bug with floating point widths has also solved this bug. Meow!

**Comment by tin-cat** [comment]
Sadly, I've re-checked this and it seems it's not really solved yet :(

**Comment by tin-cat** [comment]
Seems I finally solved this bug, it is related to a docummented jQuery bug that causes jQuery to report wrong div sizes in certain situations when the browser is zoomed in or out: https://github.com/jquery/jquery/issues/3808

Using a jQuery version >= 3.3.0 makes the bug disappear in the browsers I tested (latest versions of Chrome, Safari and Firefox in Mac Os as of august 2018).

I'm adding this information to the README file and updating the minimum required jQuery version accordingly.

Meow!
